lib/matplotlib/collections.py

```diff
@@ -2005,6 +2004,7 @@ def __init__(self, meshWidth, meshHeight, coordinates,
         self._bbox = transforms.Bbox.unit()
         self._bbox.update_from_data_xy(coordinates.reshape(
             ((meshWidth + 1) * (meshHeight + 1), 2)))
+        super().__init__(**kwargs)
```

Comment:
Why is the super init moved to the end?

Reply:
There was a test causing this to be `set_array` to be called before the initialization of  `self._meshWidth = meshWidth` and `self._meshHeight = meshHeight`.  Now that it's moved to Quadmesh I can revert back to the original changes.

Reply:
`test_pcolorfast()` calls the `set_array` method somewhere in the stack before the initialization of `self._meshWidth = meshWidth` and `self._meshHeight = meshHeight`, which then causes width and height to not be defined.